src/lib/util/test-utils.ts

```diff
@@ -20,9 +22,13 @@ export function checkWizardState(
     // Only the selected step should be selected
     expect(step.selected).toBe(index === selectedStepIndex, `expected only step ${index} to be selected`);
 
-    // All steps before the selected step need to be completed
+    // Check completed step indexes
     expect(step.completed).toBe(completedStepIndexes.includes(index),
       `expected step ${index} ${completedStepIndexes.includes(index) ? 'to be completed' : 'not to be completed'}`);
+
+    // Check step "editing" state
+    const expectation = index === selectedStepIndex && selectedStepEditing ? 'to be in editing state' : 'not to be in editing state';
+    expect(step.editing).toBe(index === selectedStepIndex && selectedStepEditing, `expected step ${index} ${expectation}`);
```

Comment:
Shouldn't this also check the constraint `if selectedStepEditing then completedStepIndexes.contains(selectedStepIndex)`?

Reply:
Good idea. Implemented in 2f354a5